src/prose/prose.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { Box } from '../box'
+import styles from './prose.module.css'
+
+type ProseProps = {
+    /**
+     * The prose content.
+     *
+     * This must consist of React nodes and elements. It is the consumer’s responsibility to
+     * generate this from other sources, such as converting raw markdown content to React elements,
+     * etc.
+     *
+     * Alternatively, you can use `<Prose dangerouslySetInnerHTML={{ __html: htmlString }}` />`
+     * instead.
+     */
+    children?: React.ReactNode
+
+    /**
+     * Sets the prose content to be raw HTML stored in a string value.
```

Comment:
I assumed we'd all know it from the name of the prop and the fact that it is the same prop used in regular HTML tags in React. Don't you think?

I even went on and check what was the jsdoc for this prop on React type definitions, because I wanted to use the same wording, and it does not have any jsdoc at all. 🤷‍♂️

Reply:
Given that this is open source, I'd opt for a higher standard and prevent people from shooting themselves in the foot. Not a strongly held opinion, though.

A different, but related point: Seeing that we're not using this in todoist-web, why do we offer this functionality in the first place?

Reply:
> A different, but related point: Seeing that we're not using this in todoist-web, why do we offer this functionality in the first place?

The goal is to use it: https://github.com/Doist/todoist-web/pull/6074

Reply:
Sure, I can add a comment about it.

And yeah, the whole idea came to be because the design team asked if we can use the same styling for markdown-generated text in Todoist (tasks, comments), and also in contentful-generated content in Todoist integration settings.

Reply:
> Can we add a warning about security implications here, please? :)

Done. It does look better now. Thanks for insisting on this.

https://github.com/Doist/reactist/blob/4f8eb94578f3121a4449d8447701a81ecd22721e/src/prose/prose.tsx#L18-L29